phpunit/block-supports/background-test.php

```diff
@@ -143,7 +134,7 @@ public function data_background_block_support() {
 						'url' => 'https://example.com/image.jpg',
 					),
 				),
-				'expected_wrapper'    => '<div class="has-background" style="background-image:url(' . $apos . 'https://example.com/image.jpg' . $apos . ');background-size:cover;">Content</div>',
+				'expected_wrapper'    => '<div class="has-background" style="background-image:url(&#039;https://example.com/image.jpg&#039;);background-size:cover;">Content</div>',
```

Comment:
These could become the `'` character for more legible tests:

```suggestion
				'expected_wrapper'    => '<div class="has-background" style="background-image:url(\'https://example.com/image.jpg\');background-size:cover;">Content</div>',
```